sqlite.go

```diff
@@ -363,8 +385,31 @@ func (r *rows) Next(dest []driver.Value) (err error) {
 				if err != nil {
 					return err
 				}
-
+				cty, err := r.columnDeclType(i)
```

Comment:
This one makes actually sense.

Atleast if we want to handle int->boolean conversion by default,
which seems like a straightforward thing to do?